README.md

```diff
@@ -1 +1,120 @@
-# 522-wine-quality-32
\ No newline at end of file
+# Reproducible Computational Environment
+
+This project ensures a reproducible computational environment using Conda. Follow the steps below to set up the environment for this project.
+
+## Prerequisites
+
+1. Install Miniconda or Anaconda.
+2. Clone this repository:
+
+    ```bash
+    git clone https://github.com/UBC-MDS/522-wine-quality-32.git
+    cd 522-wine-quality-32
+    ```
+
+## Setting Up the Environment
+
+### Option 1: Using `environment.yaml`
+
+This is the recommended method to set up the environment.
+
+1. Create the Conda environment:
+
+    ```bash
+    conda env create -f environment.yaml
+    ```
+
+2. Activate the environment:
+
+    ```bash
+    conda activate 522_milestone_env
+    ```
+
+3. Verify the environment setup:
+
+    ```bash
+    python -c "import pandas as pd; print('Environment set up successfully!')"
+    ```
+
+### Option 2: Using Platform-Specific Lock Files
+
+If you want to ensure reproducibility across different operating systems, use platform-specific lock files.
+
+1. Install `conda-lock`:
+
+    ```bash
+    pip install conda-lock
+    ```
+
+2. Create the environment using the lock file for your platform:
```

Comment:
We might want to make this unform across platforms by using `conda-lock` file, right?